raw_package/base.template.html

```diff
@@ -20,18 +20,13 @@
       as="font"
       crossorigin="anonymous"
     />
-    <link
-      rel="stylesheet"
-      href="{{ get_static_file_url('css/vendor/materialize/materialize.min.css') }}"
-    />
     <link
       rel="stylesheet"
       href="{{ get_static_file_url('css/esphome-2.css') }}"
     />
   </head>
 
   <body>
-    {% block main %}{% end %}
-    {% block footer %}{% end %}
+    {% block main %}{% end %} {% block footer %}{% end %}
```

Comment:
Please revert this change.

Reply:
prettier formatted this. Give me a sec and I revert it.

Reply:
done